sonar-plugin-api/src/main/java/org/sonar/api/web/WidgetCategory.java

```diff
@@ -26,7 +26,9 @@
 
 /**
  * @since 2.4
+ * @deprecated since 6.2, as dashboards have been removed
  */
+@Deprecated
 @Retention(RetentionPolicy.RUNTIME)
 @Target(ElementType.TYPE)
 public @interface WidgetCategory {
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Do not forget to remove this deprecated code someday. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=squid%3AS1133)